source/content/modules-plugins-known-issues.md

```diff
@@ -690,6 +690,19 @@ An alternative solution is to [create a symbolic link](/assuming-write-access/#c
 
 **Solution:** A [user patch](https://gist.github.com/kshaner/7fcbc7e3e967c5694fd38638bff1cc17/revisions) has been [submitted](https://wordpress.org/support/topic/absolute-path-causes-issues-on-environments-with-multiple-containers/) to the plugin maintainers.
 
+<hr/>
+
+### [PolyLang](https://wordpress.org/plugins/nextgen-gallery/)
+**Issue**: The plugin adds a cache-busting cookie (ex. `pll_language=en` ) for each request.
+
+**Solution**: A constant setting is available in `wp-config.php` to remove or rename the cookie:
+
+```php:title=wp-config.php
+define('PLL_COOKIE', false)
+```
+
+The option `PLL_COOKIE` defaults to `pll_polylang`. This defines the name of the cookie used by Polylang to store the visitor's language. When `PLL_COOKIE` is set to false, Polylang does not set any cookie. Be aware that in this case, not everything will work correctly. For example, the login page will not translate.
```

Comment:
> Be aware that in this case, not everything will work correctly. For example, the login page will not translate.

@eabquina is this to say that this code change will fix caching, but break the use of the plugin? Also, have we reported this issue to the plugin authors?

Reply:
@alexfornuto the code change will fix caching because it will remove the cache-busting cookie. The note says that the plugin will generally work. We did not list every single part of the site or potential translated pages that might use the cookies. This is left for the customer/user to figure out. 

A potential scenario is to rename the cookie to follow the STYXKEY pattern, but yet to be proven by another site who might benefit for it. The related customer for this report didn't need to use the cookies in their case. 

Actually, I saw this issue reported before by others and was resolved with the instruction added in the note. See https://github.com/polylang/polylang/issues/248